module/VuFindSearch/src/VuFindSearch/Backend/Blender/Response/Json/RecordCollection.php

```diff
@@ -185,6 +185,27 @@ public function setSourceIdentifier($identifier)
         // collectBackendRecords below.
     }
 
+    /**
+     * Get delimiter for the given facet field
+     *
+     * @param string $facet Facet field
+     *
+     * @return string
+     */
+    public function getFacetDelimiter(string $facet): string
+    {
+        foreach ($this->config->Advanced_Settings->delimited_facets ?? []
+            as $facet
+        ) {
+            $parts = explode('|', $facet);
+            if ('blender_backend' === $parts[0]) {
```

Comment:
Why is blender_backend hard-coded here? Based on the comment at the top, this seems like it's meant to be a generic method, but it seems to be very specifically tied to blender_backend at this point.

Reply:
Also, why are you passing in a $facet parameter and then immediately overriding it with the foreach loop?

Reply:
Thanks, makes much more sense now. :-)